app/components/views/TicketsPage/PurchaseTab/Tickets.js

```diff
@@ -7,11 +7,12 @@ import { ShowWarning, Subtitle } from "shared";
 import "style/PurchaseTickets.less";
 import { InfoDocModalButton } from "buttons";
 import UnsignedTickets from "./UnsignedTickets";
+import modalStyle from "../../../modals/Modals.module.css";
```

Comment:
Please create a module for TicketsPage and use it here, this import does make sense here